modules/codegen/src/main/scala/com/twilio/guardrail/generators/AkkaHttpClientGenerator.scala

```diff
@@ -259,7 +259,9 @@ object AkkaHttpClientGenerator {
           // Placeholder for when more functions get logging
           _ <- Target.pure(())
 
-          produces = operation.produces.toList.flatMap(RouteMeta.ContentType.unapply(_))
+          produces = NonEmptyList
+            .fromList(operation.produces.toList.flatMap(RouteMeta.ContentType.unapply(_)))
+            .getOrElse(NonEmptyList.one(RouteMeta.ApplicationJson))
```

Comment:
No change needed, just something I've had on my mind on and off: should we be basing a default consumes/produces based on the type of the body?  Like if it's `type: string`, maybe text/plain.  If it's a `$ref` to some `type: object`, JSON... etc.

Reply:
I didn't do it here, but I'm leaning more towards just straight up failing unless the produces/consumes is supplied. The use case I'm weighing against is if the user doesn't have access to modify the OpenAPI specification file they are generating from.